src/components/footer.js

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+
+const Footer = () => (
+  <footer
+    style={{
+      background: `white`,
+      position: `absolute`,
+      bottom: `0`,
+      width: `100%`,
+      zIndex: `-1`,
+    }}
+  >
+    <div
+      style={{
+        margin: `0 auto`,
+        maxWidth: 960,
+        padding: `1rem 1rem`,
+      }}
+    >
+      <span style={{ margin: `0`, fontSize: `0.7rem` }}>
+        <a href="https://thenounproject.com/term/montana/468266">“Montana”</a>
+        {" icon by Ted Grajeda from "}
+        <a href="https://thenounproject.com/">the Noun project</a>.
```

Comment:
Pretty sure our plan is to pay the $2.99 or whatever so that we don't have to attribute it, so we won't need this.

Reply:
Sorry @Turkinolith, I'm a dumbie and didn't even look into attribution. George bought the license, so this file can be removed.

Reply:
Okay, will submit a new commit to this PR without that bit.